pkgs/top-level/darwin-packages.nix

```diff
@@ -66,12 +66,7 @@ makeScopeWithSplicing' {
   # macOS 12.3 SDK
   apple_sdk_12_3 = pkgs.callPackage ../os-specific/darwin/apple-sdk-12.3 { };
 
-  # Pick an SDK
-  apple_sdk = {
-    "10.12" = apple_sdk_10_12;
-    "11.0" = apple_sdk_11_0;
-  }.${stdenv.hostPlatform.darwinSdkVersion}
-  or (throw "Unsupported sdk: ${stdenv.hostPlatform.darwinSdkVersion}");
+  apple_sdk = apple_sdk_11_0;
```

Comment:
does this need to be coupled to `darwinSdkVersion` with an assert or something? there wasn't one before I added it and seemed like it just broke things like android and iOS eval.

Reply:
Hmm, since this is now just a compatibility alias that are going away by the next time this changes in 25.11, it seems fine? I struggle to imagine how it could break Android eval if it’s not even accessing the platform.

I thought about adding an assert to `apple-sdk` to make sure that the platform fields are updated to the right minor version when the SDK is bumped, but I couldn’t quite think of a properly elegant way to do it, and since our minimum supported macOS release is going to continue to be years old it seems unlikely they’ll get minor SDK updates very often.

Reply:
oh yeah, I forget `apple-sdk` is the thing now -- and since that uses `darwinSdkVersion` I guess the assert I was looking for is not needed.